irony-cdb.el

```diff
@@ -63,6 +63,20 @@ for files that it cannot handle."
   :type '(repeat function)
   :group 'irony-cdb)
 
+(defcustom irony-cdb-search-directory-list '("." "build")
```

Comment:
This is from me, copy-pasted from LLVM cmake build, I would not go as far as to say one project makes it "official". But yes this is a sound guess. :)

For example, on QtCreator, the build is often like this by default I believe:

* foo/project/CMakeLists.txt
* foo/project-qtcreator-build

Something along these lines.